src/pipeline/activator/requirements-conda.txt

```diff
@@ -8,4 +8,6 @@ tqdm==4.61.2
 h5py==3.3.0
 pyproj==3.1.0
 shapely==1.7.1
-rasterio==1.2.1
\ No newline at end of file
+rasterio==1.2.1
+pytorch-cpu=1.9.0
+torchvision-cpu=0.9.1
```

Comment:
I'm wondering, how heavy this deps are? If they are not let's keep them as-is, if they are: would it make sense to have a separate docker image?